force-app/main/default/prompts/OrderBuilder.prompt-meta.xml

```diff
@@ -0,0 +1,118 @@
+        <delayDays>1</delayDays>
+        <description
+        >Use the Order Builder to learn more about component composition.</description>
+        <dismissButtonLabel>Close</dismissButtonLabel>
+        <displayPosition>TopCenter</displayPosition>
+        <displayType>FloatingPanel</displayType>
+        <isPublished>true</isPublished>
+        <masterLabel>Order Builder Step 1</masterLabel>
+        <publishedDate>2020-10-09</publishedDate>
+        <shouldDisplayActionButton>false</shouldDisplayActionButton>
+        <shouldIgnoreGlobalDelay>true</shouldIgnoreGlobalDelay>
+        <startDate>2020-10-09</startDate>
+        <stepNumber>1</stepNumber>
+        <targetAppDeveloperName>EBikes</targetAppDeveloperName>
+        <targetPageKey1>Order__c</targetPageKey1>
+        <targetPageKey2>view</targetPageKey2>
+        <targetPageType>standard__recordPage</targetPageType>
+        <timesToDisplay>1</timesToDisplay>
+        <title>Welcome to the Order Builder</title>
+        <uiFormulaRule>
+            <criteria>
+                <leftValue
+                >{!$Permission.StandardPermission.AccessWalkthroughs}</leftValue>
+                <operator>EQUAL</operator>
+                <rightValue>TRUE</rightValue>
+            </criteria>
+        </uiFormulaRule>
+        <userAccess>SpecificPermissions</userAccess>
+        <userProfileAccess>Everyone</userProfileAccess>
+        <versionNumber>1</versionNumber>
+    </promptVersions>
+    <promptVersions>
+        <body
+        >Here you can see a list of available products to add to each order as well as search for products that might not be in the list. &lt;br&gt; &lt;br&gt; Try searching for an &quot;Electra X1&quot; and see the results as they&apos;re updated.</body>
+        <dismissButtonLabel>Close</dismissButtonLabel>
+        <displayPosition>BottomRight</displayPosition>
+        <displayType>FloatingPanel</displayType>
+        <isPublished>false</isPublished>
+        <masterLabel>Order Builder Step 2</masterLabel>
+        <shouldDisplayActionButton>false</shouldDisplayActionButton>
+        <shouldIgnoreGlobalDelay>false</shouldIgnoreGlobalDelay>
+        <stepNumber>2</stepNumber>
+        <targetAppDeveloperName>EBikes</targetAppDeveloperName>
+        <targetPageKey1>Order__c</targetPageKey1>
+        <targetPageKey2>view</targetPageKey2>
+        <targetPageType>standard__recordPage</targetPageType>
+        <title>Product List</title>
+        <userAccess>Everyone</userAccess>
+        <userProfileAccess>Everyone</userProfileAccess>
+        <versionNumber>1</versionNumber>
+    </promptVersions>
+    <promptVersions>
+        <body
+        >You can drag products from the list into the basket. As you add products, the totals will update and relate the records to the order. &lt;br&gt;&lt;br&gt; The drop is handled using the standard &lt;a href=&quot;#=https://mzl.la/36N7i4c&quot;&gt;Drag &amp; Drop APIs&lt;/a&gt;.</body>
```

Comment:
Something's wrong with the link here. I can't use it, it closes the dialog box.

Reply:
I've removed this as it doesn't seem we can hyperlink outside of salesforce.